.github/workflows/publish-conda.yml

```diff
@@ -20,13 +19,39 @@ jobs:
         with:
           token: ${{ secrets.GIT_TOKEN }}
 
-      - name: Publish to Conda
-        uses: marek-mihok/publish_conda_package_action@v1.0.1
+      - uses: actions/setup-go@v1
         with:
-          CondaDir: 'py/h2o_wave/conda'
-          Platforms: 'noarch'
-          CondaUsername: ${{ secrets.CONDA_USERNAME }}
-          CondaPassword: ${{ secrets.CONDA_PASSWORD }}
+          go-version: '1.19.4'
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Setup
+        run: make setup
+
+      - uses: r-lib/actions/setup-r@v2
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          working-directory: ./r
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-activate-base: false
+          activate-environment: ""
+          conda-build-version: '24.3.0'
+
+      - name: Build Release
+        run: make release
         env:
           # VERSION clashes with conda build. Use PKG_VERSION instead.
           PKG_VERSION: ${{ env.VERSION }}
+          NODE_OPTIONS: '--max-old-space-size=8192'
+
+      - name: Publish to Conda
+        uses: marek-mihok/upload_to_anaconda_cloud_action@v1.0.0
```

Comment:
What was the reason for switching from the previous conda publish action?

Reply:
The previous action was building a package (the only input was path to conda dir with yaml files), converting it for every platform and uploading it to anaconda cloud. But you couldn't have control over moving files before each build.

That's why I added a workflow `Setup Miniconda` step and then the packages are built during `make release`. The new action then only uploads packages to Anaconda cloud.

I can also think of another solution and that is running the previous gh action multiple times with moving the files before each run. But this way if something fails, only part of the packages will be uploaded to Anaconda cloud.

Reply:
Sounds reasonable. Thanks!